settings.gradle.kts

```diff
@@ -9,3 +9,11 @@ include("grpc-api")
 buildCache {
     local.isEnabled = true
 }
+include("domainservice:stock-record")
+findProject(":domainservice:stock-record")?.name = "stock-record"
```

Comment:
why do you need to rename modules?

Reply:
It was generated by IDEA. I removed each module renaming